File: Examples/VisualBasic/CellsHelperClass/NameToIndex.vb
Imports System

Namespace CellsHelperClass
    Public Class NameToIndex
        Public Shared Sub Run()
            ' ExStart:1
            Dim name As String = "C4"
            Dim row As Integer = Nothing
            Dim column As Integer = Nothing
            Aspose.Cells.CellsHelper.CellNameToIndex(name, row, column)
            Console.WriteLine("Row: {0}, Column: {1}", row, column)
            ' ExEnd:1

        End Sub
    End Class
End Namespace

File: Examples/CSharp/CellsHelper/NameToIndex.cs
using System;

namespace Aspose.Cells.Examples.CSharp._CellsHelper
{
    public class NameToIndex
    {
        public static void Run()
        {
            string name = "C4";
            int row;
            int column;
            Aspose.Cells.CellsHelper.CellNameToIndex(name, out row, out column);
            Console.WriteLine("Row: {0}, Column: {1}", row, column);

            Console.WriteLine("NameToIndex executed successfully.");
        }
    }
}
